src/main/webapp/app/overview/course-overview.scss

```diff
@@ -169,3 +169,47 @@
 .course-exercises-search {
     width: 100%;
 }
+
+.shakeIcon {
+    /* Start the shake animation and make the animation last for 0.5 seconds */
+    animation: shake 0.2s;
+
+    /* When the animation is finished, start again */
+    animation-iteration-count: initial;
```

Comment:
I am not an CSS expert but `initial` is equal to 1 here, right? So it shakes once or am I thinking about this wrong? I am asking because the comment would imply it shakes forever